README.md

```diff
@@ -18,6 +18,8 @@ A VS Code extension that allows you to use the browser's Elements tool from with
 * Side Bar view for listing all the debuggable targets, including tabs, extensions, service workers, etc.
 * Fully featured Elements tool with views for HTML, CSS, accessibility and more.
 * Screen-casting feature to allow you to see your page without leaving VSCode.
+* Go directly to the line/column for source files in your workspace when clicking on a link inside the Elements tool.
```

Comment:
Or clicking on an individual CSS rule, right? (Or is that only once the file is opened?)